membership-attribute-service/app/models/Attributes.scala

```diff
@@ -1,21 +1,31 @@
 package models
 
+import json._
 import play.api.libs.json._
 import play.api.mvc.Result
 import play.api.mvc.Results.Ok
 
 import scala.language.implicitConversions
 
+case class ContentAccess(member: Boolean, paidMember: Boolean)
+
+object ContentAccess {
+  implicit val jsWrite = Json.writes[ContentAccess]
+}
+
 case class Attributes(userId: String, tier: String, membershipNumber: Option[String]) {
   require(tier.nonEmpty)
   require(userId.nonEmpty)
 
   lazy val isFriendTier = tier.equalsIgnoreCase("friend")
   lazy val isPaidTier = !isFriendTier
+
+  lazy val contentAccess = ContentAccess(member = true, paidMember = isPaidTier) // we want to include staff!
 }
 
 object Attributes {
-  implicit val jsWrite = Json.writes[Attributes]
+  implicit val jsWrite: Writes[Attributes] =
+    Json.writes[Attributes].asInstanceOf[OWrites[Attributes]].addField("contentAccess", _.contentAccess)
```

Comment:
This annoying cast is necessary because currently the `Json.writes` macro has a declared return type of `Writes` rather than `OWrites` (a type which guarantees to return a `JsObject`, rather than just a `JsValue`).

This is actually fixed in a forthcoming release of Play: https://github.com/playframework/playframework/pull/4772